src/drive/store/configureStore.js

```diff
@@ -20,7 +20,7 @@ const configureStore = (client, t, initialState = {}) => {
   if (shouldEnableTracking() && getTracker()) {
     middlewares.push(createTrackerMiddleware())
   }
-  middlewares.push(createLogger(loggerOptions()))
```

Comment:
is this ok? `createLogger` accepts a function as argument or did you forgot to include a change to `loggerOptions` below?